keras_spiking/tests/test_model_energy.py

```diff
@@ -100,9 +103,10 @@ def test_layer_repeat(rng):
     model1 = tf.keras.Model(inp, [layer(inp)])
     model2 = tf.keras.Model(inp, [layer(layer(inp))])
 
-    energy1 = ModelEnergy(model1, input_data=input_data)
-    energy2 = ModelEnergy(model2, input_data=input_data)
-    for stat in ["synops", "neurons"]:
+    with tf.device("CPU"):
```

Comment:
I assigned these to the CPU so that I could do other GPU work at the same time. And then I thought "well might as well leave it like that", since there isn't really any advantage to running these tests on the GPU. But we could also just leave it unspecified.

Reply:
Yeah, makes sense. The only time I'd run on GPU is if `input_data` was big or the model was big, but neither is the case. The only concern I can think of is that a user looks at the test and thinks it HAS to run on CPU, but I don't think many users are looking at the tests, and hopefully those who do are smart enough to figure this out.

Reply:
Maybe we should leave a comment, though, so we don't think it's an integral part of the test? We don't want to think in the future that it doesn't work on GPU or something like that.